velox/exec/DistinctAggregations.cpp

```diff
@@ -27,10 +27,11 @@ class TypedDistinctAggregations : public DistinctAggregations {
       std::vector<AggregateInfo*> aggregates,
       const RowTypePtr& inputType,
       memory::MemoryPool* pool)
-      : aggregates_{std::move(aggregates)},
-        input_{aggregates_[0]->inputs[0]},
-        inputType_{inputType->childAt(input_)},
-        pool_{pool} {}
+      : isMultichannelDistinct_(aggregates[0]->inputs.size() > 1),
```

Comment:
It looks like we don't need this member variable. We can use inputs_.size() == 1 instead or add a helper method isSingleInput().